src/components/MemeGenerator.js

```diff
@@ -31,6 +32,15 @@ class MemeGenerator extends React.Component {
 			});
 	}
 
+  getMyMemes = () => {
+    axios.get("http://localhost:3001/api/memes").then(response => {
+      const myMemes = response.data
+      this.setState({
+        mySavedMemes: myMemes
+      })
+    })
+  }
```

Comment:
Great work - just fyi - we can address this in a future pull request:

In the `getImages()`method we're using `fetch` as the http client - but here in the `getMyMemes()` we're using `axios`. Either way is fine - but we should probably just pick one.

Also, we'll need to make the server location (`http://localhost:3001`) dynamic - I usually do this by putting it into an environment variable like `${process.env.REACT_APP_API_SERVER}/api/memes`. [see here](https://create-react-app.dev/docs/adding-custom-environment-variables/#adding-development-environment-variables-in-env)